### ISSUE TITLE
[Bug] Non-default abilities lack sound & particles

### ISSUE THREAD

**Issue by ActionAlligator** [darklordabc/Legends-of-Dota-Redux#1548]
Anytime I add an ability outside the default hero's selection, that ability lacks any sound, particles, or animations. Have tested both on local server and Valve's servers. Chain Lightning and Counter Helix are the ones that come to mind, so I'm not sure if this is just some abilities or all, or even if this is just unique to me.

**Comment by Wouterz90** [comment]
Is this since the latest update?

**Comment by arkydon** [comment]
Yeah I'm facing this issue too. It started happening after one LOD update sometime after 7.20 was released.

**Comment by Wouterz90** [comment]
Okay. I know what caused it then.
